### PR TITLE
New package: tap-plugins-1.0.0

### PR DESCRIPTION
**Pull request by laserbat**

### CODE REVIEW

File: srcpkgs/tap-plugins/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+
+do_configure() {
+	sed 's|/usr/local/|$(DESTDIR)/usr/|' -i Makefile
+}
+
+do_build() {
+	make ${makejobs}
```
Comment (Johnnynator) [inline srcpkgs/tap-plugins/template]: Can you try setting `make_use_env=yes` globally and remove the custom `do_build`

Comment (laserbat) [reply srcpkgs/tap-plugins/template]: Done, it works! Thank you :)

**Comment by Johnnynator** [comment]
Cross builds should be fixable by adding `make_build_args="CC=${CC}"

**Comment by laserbat** [comment]
@Johnnynator added that parameter. Let's see if it compiles! Thank you

File: srcpkgs/tap-plugins/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+make_use_env=yes
+make_build_args="CC=${CC}"
```
Comment (Johnnynator) [inline srcpkgs/tap-plugins/template]: Sry I forgot, this does not work properly, it needs to be set inside of a function, putting it into `pre_build` does work

File: srcpkgs/tap-plugins/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+make_use_env=yes
```
Comment (Johnnynator) [inline srcpkgs/tap-plugins/template]: Move `make_use_env` directly under `build_style

File: srcpkgs/tap-plugins/template (hunk 2 above)
Comment (laserbat) [reply srcpkgs/tap-plugins/template]: Okay, fixed!

File: srcpkgs/tap-plugins/template (hunk 3 above)
Comment (laserbat) [reply srcpkgs/tap-plugins/template]: Done!

File: srcpkgs/tap-plugins/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+
+pre_build() {
+    make_build_args="CC=${CC}"
```
Comment (Johnnynator) [inline srcpkgs/tap-plugins/template]: Intend with tabs. Otherwise it should be fine now.

Comment (laserbat) [reply srcpkgs/tap-plugins/template]: done

File: srcpkgs/tap-plugins/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+
+pre_build() {
+	make_build_args="CC=${CC} LD=$(LD)"
```
Comment (Johnnynator) [inline srcpkgs/tap-plugins/template]: `LD=$(LD)` Will try to execute LD, you probably want `LD=${LD}`.

Comment (laserbat) [reply srcpkgs/tap-plugins/template]: My bad, missed this typo! Fixed

**Pull request merged**